client/tests/simulator_tests.rs

```diff
@@ -51,9 +52,35 @@ fn get_started_tpm() -> (TpmSim, TcpTpm) {
 }
 
 #[test]
-fn test_get_manufacturer_id() {
+fn test_get_capability_manufacturer_id() {
     let (_sim_lifeline, mut tpm) = get_started_tpm();
-    assert!(get_manufacturer_id(&mut tpm).is_ok());
+
+    let mut expected = TpmlTaggedTpmProperty {
+        count: 1,
+        tpm_property: [TpmsTaggedProperty::default(); 127],
+    };
+
+    expected.tpm_property[0] = TpmsTaggedProperty {
+        property: TpmPt::Manufacturer,
+        value: 0x4D534654,
+    };
+
+    let command: GetCapabilityCmd = GetCapabilityCmd {
```

Comment:
Nit: You don't need to re-state `GetCapabilityCmd` here, the type is clearly known by the rhs of the assignment operator.

I'd just:

```rs
let command = GetCapabilityCmd {
    ...
};
```